src/main.js

```diff
@@ -7,8 +7,11 @@ import axios from "axios";
 import { ValidationProvider, setInteractionMode } from "vee-validate";
 import "./components/validationRules";
 import authService from "./services/authService";
+import subscriptionService from "./services/subscriptionService";
```

Comment:
i saw somewhere that you are importing mapService to each component. Import here and give access to it globally by $mapService